hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java

```diff
@@ -103,6 +103,11 @@ public class MerkleHederaState extends PartialNaryMerkleInternal
      */
     private final Map<String, Map<String, StateMetadata<?, ?>>> services = new HashMap<>();
 
+    // Default constructor provided for ConstructableRegistry, TO BE REMOVED ASAP
+    public MerkleHederaState() {
+
+    }
```

Comment:
Do we get compilation error without this constructor ?

Reply:
Understood from meeting.

Reply:
No, it is all reflection by the platform during startup and deserialization.